media-video/vdr/vdr-2.4.4.ebuild

```diff
@@ -0,0 +1,203 @@
+		[[ -d ${i} ]] || continue
+		if [[ -f ${i}/linux/dvb/dmx.h ]]; then
+			einfo "Found DVB header files in ${i}"
+			DVBDIR=${i}
+			break
+		fi
+	done
+
+	# checking for s2api headers
+	local api_version
+	api_version=$(awk -F' ' '/define DVB_API_VERSION / {print $3}' "${DVBDIR}"/linux/dvb/version.h)
+	api_version=${api_version}*$(awk -F' ' '/define DVB_API_VERSION_MINOR / {print $3}' "${DVBDIR}"/linux/dvb/version.h)
+
+	if [[ ${api_version:-0} -lt 5*3 ]]; then
+		eerror "DVB header files do not contain s2api support or too old for ${P}"
+		eerror "You cannot compile VDR against old dvb-header"
+		die "DVB headers too old"
+	fi
+
+	cat > Make.config <<-EOT || die "cannot write to Make.config"
+		#
+		# Generated by ebuild ${PF}
+		#
+		PREFIX			= /usr
+		DVBDIR			= ${DVBDIR}
+		PLUGINLIBDIR	= ${PLUGIN_LIBDIR}
+		CONFDIR			= ${CONF_DIR}
+		ARGSDIR			= \$(CONFDIR)/conf.d
+		VIDEODIR		= ${VIDEO_DIR}
+		LOCDIR			= \$(PREFIX)/share/locale
+		INCDIR			= \$(PREFIX)/include
+
+		DEFINES			+= -DCONFDIR=\"\$(CONFDIR)\"
+		INCLUDES		+= -I\$(DVBDIR)
+
+		# >=vdr-1.7.36-r1; parameter only used for compiletime on vdr
+		# PLUGINLIBDIR (plugin Makefile old) = LIBDIR (plugin Makefile new)
+		LIBDIR			= ${PLUGIN_LIBDIR}
+		PCDIR			= /usr/$(get_libdir)/pkgconfig
+
+	EOT
+	eend 0
+
+	eapply "${FILESDIR}/vdr-2.4.1_gentoo.patch"
+	eapply "${FILESDIR}/vdr-2.2.0-jpeg-9c.patch"
+	use demoplugins || eapply "${FILESDIR}/vdr-2.4_remove_plugins.patch"
+	eapply "${FILESDIR}/${P}_makefile-variables.patch"
+
+	use naludump && eapply "${DISTDIR}/${PN}-2.1.5-naludump-0.1.diff"
+	use permashift && eapply "${FILESDIR}/vdr-2.4.4-patch-for-permashift.diff"
+	use pinplugin && eapply "${FILESDIR}/${P}_pinplugin.patch"
+	use ttxtsubs && eapply "${WORKDIR}/${P}_ttxtsubs.patch"
+	use menuorg && eapply "${DISTDIR}/vdr-menuorg-2.3.x.diff"
+	use mainmenuhooks && eapply "${WORKDIR}/vdr-2.4.1_mainmenuhook-1.0.1.patch"
+
+	add_cap CAP_UTF8 \
+		CAP_IRCTRL_RUNTIME_PARAM \
+		CAP_VFAT_RUNTIME_PARAM \
+		CAP_CHUID \
+		CAP_SHUTDOWN_AUTO_RETRY
+
+	echo -e ${CAPS} > "${CAP_FILE}" || die "cannot write to CAP_FILE"
+
+	# LINGUAS support
+	einfo "\n \t VDR supports the LINGUAS values"
+
+	lang_po
+
+	einfo "\t Please set one of this values in your sytem make.conf"
+	einfo "\t LINGUAS=\"${LING_PO}\"\n"
+
+	if [[ -z ${LINGUAS} ]]; then
+		einfo "\n \t No values in LINGUAS="
+		einfo "\t You will get only english text on OSD \n"
+	fi
+
+	strip-linguas ${LING_PO} en
+
+	default
+}
+
+src_install() {
+	# trick the makefile to not create a VIDEODIR by supplying it with an
+	# existing directory
+	emake VIDEODIR="/" DESTDIR="${ED}" install
+
+	keepdir "${PLUGIN_LIBDIR}"
+
+	# backup for plugins they don't be able to create this dir
+	keepdir "${CONF_DIR}/plugins"
+
+	if use html; then
+		local HTML_DOCS=( *.html )
+	fi
+	local DOCS=( MANUAL INSTALL README* HISTORY CONTRIBUTORS UPDATE-2* )
+	einstalldocs
+
+	insinto /usr/share/vdr
+	doins "${CAP_FILE}"
+
+	fowners vdr:vdr "${CONF_DIR}" -R
```

Comment:
Hmm... I would like to add acct-*/vdr but.... this is a longer story.
I reserved the UID+GID for vdr long time ago through the dev-ML and prepared acct-*/vdr for it. vdr traditionally uses HOME=/var/vdr which is not FHS conform, but it is tradition. I had long discussions with the last vdr maintainter "hd_brummy" (who is absent again and doesnt even give a short "leave me alone" reply) about this years ago. But on his last "work phase" he already prepared the acct-user/vdr in his overlay with a different HOME=/var/lib/vdr  (you may look [here](https://gitweb.gentoo.org/proj/vdr/devel.git/tree/acct-user/vdr/vdr-0.ebuild) which is fine for me too. You see code reflecting the upcoming change in querying current $HOME of user vdr.

Unfortunately, hd_brummy invented a USE logic for group memberships  as a user convenience solution for some vdr plugins. I have not completed thinking about that, but I expect  that will need new revisions of the plugins which may need these extra groups.

Long story short, I definitely want acct-* packages, but I hope we can commit this one here without them.

Or - if you agree - I add simplified versions of acct-*/vdr into this version bump (in separate commits).
Let me know....

Reply:
I'd really like to have them as separate commits in here, because it looks like vdr doesnt' launch unless I **manually** add the group and add vdr to video group. But then again I wouldn't wish to make this too huge with all those plugin revbumps. Can you show me an example of a current plugin ebuild that handles this? You maybe could be able to do bash magic in the acct ebuild, which sets `/var/vdr` or `/var/lib/vdr` depending if that directory exists in the live filesystem or not. Not something I'd encourage though.

Reply:
As a prerequisite, you have to know that user+group "vdr" are created in media-tv/gentoo-vdr-scripts. media-video/vdr depends on media-tv/gentoo-vdr-scripts. So if you did not install media-tv/gentoo-vdr-scripts, it may come to problems. 

And, a known issue for me is that on systemd installations there are not all file+dir owners are set correctly. an updated media-tv/gentoo-vdr-scripts will fix this, but when I revbump media-tv/gentoo-vdr-scripts without acct-*/vdr then we have again the discussion with acct-*/vdr. So please ignore this for now.

IMHO when installing acct-user/something, a previously existing user and his setting is untouched. So IMHO we need no magic when current ~vdr is different from /var/lib/vdr. But all subseqent packages must respect ~vdr like `VIDEO_DIR="$(egethome vdr)/video"`. Agree?

Reply:
So the plan would be to revbump all vdr-related packages at once, using the new acct- method? I'm fine with that, as long as it's resolved somehow (sorry this is very confusing trying to read). We can also write a warning to users utilizing `REPLACING_VERSION` variable in ebuild.

I'm fine with pushing this if you're up to resolving the vdr issue. But please remove the `html` USE flag and install it unconditionally.